plugins/base/src/main/kotlin/parsers/Parser.kt

```diff
@@ -49,10 +51,51 @@ abstract class Parser {
 
     private fun jkdocToListOfPairs(javadoc: String): List<Pair<String, String>> =
         "description $javadoc"
-            .split("\n@")
+            .splitIgnoredInsideBackticks("\n@")
             .map { content ->
                 val contentWithEscapedAts = content.replace("\\@", "@")
                 val (tag, body) = contentWithEscapedAts.split(" ", limit = 2)
                 tag to body
             }
+
+    private fun CharSequence.splitIgnoredInsideBackticks(delimiter: String): List<String> {
```

Comment:
You were right when you said it's not a simple split anymore. Suggestions:

1. the name can be changed to something like `parseNewLineTagSections`
2. `delimiter` argument can be dropped, it's highly unlikely that there will be some other char, not `"\n@"`. Moreover, the whole idea of ignoring backticks loses it's meaning if another delimiter is used, and then simple `split` could be used instead.

Reply:
On the other hand, another delimiter can be used to parse, e.g. a md unordered list.